### PR TITLE
[OSCD] sysmon_tttracer_mod_load.yml added

### PR DESCRIPTION
**Pull request by esebese**
Both parent execution and module loads implemented in a global rule

### CODE REVIEW

File: rules/windows/image_load/sysmon_tttracer_mod_load.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+action: global
+title: Time Travel Debugging Utility Usage
+id: e76c8240-d68f-4773-8880-5c6f63595aaf
+description: Detects usage of Time Travel Debugging Utility. Adversaries can execute malicious processes and dump processes, such as lsass.exe, via tttracer.exe.
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Tttracer/
+    - https://twitter.com/mattifestation/status/1196390321783025666
+    - https://twitter.com/oulusoyum/status/1191329746069655553
+author: 'Ensar Şamil, @sblmsrsn, OSCD Community'
```
Comment (yugoslavskiy) [inline rules/windows/image_load/sysmon_tttracer_mod_load.yml:9]: Hello @esebese !

Thank you for your contribution (: 

could you please change "OSCD Community" to "oscd.ommunity" or "@oscd_initiative"?
the thing is that oscd is an initiative, not a community.

the only place where you can see the "community" word near to the "oscd" word is the domain name.
but the only reason I've chosen this domain is that it was cheap and kinda matching the idea — it is a community activity of course, but the global infosec community, not the "oscd community".

sorry for the inconvenience.

Comment (esebese) [reply rules/windows/image_load/sysmon_tttracer_mod_load.yml:9]: You got me when you said cheap (: I will change it.

**Comment by yugoslavskiy** [comment]
Hello @esebese! Thank you very much for your contribution! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**